config/externalnamenottested.go

```diff
@@ -751,4 +751,48 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	// aws_storagegateway_working_storage can be imported by using the gateway Amazon Resource Name (ARN) and local disk identifier separated with a colon (:)
 	// Example: arn:aws:storagegateway:us-east-1:123456789012:gateway/sgw-12345678:pci-0000:03:00.0-scsi-0:0:0:0
 	"aws_storagegateway_working_storage": config.TemplatedStringAsIdentifier("", "{{ .parameters.gateway_arn }}:{{ .parameters.disk_id }}"),
+
+	// networkfirewall
+	//
+	// Network Firewall Policies can be imported using their ARN
+	// Example: arn:aws:network-firewall:us-west-1:123456789012:firewall-policy/example
+	"aws_networkfirewall_firewall_policy": config.TemplatedStringAsIdentifier("", "arn:aws:network-firewall:{{ .parameters.region }}:{{ .setup.client_metadata.account_id }}:firewall-policy/{{ .external_name }}"),
```

Comment:
We need to use `name` as the `externalNamePath` parameter of the function. 

```suggestion
	"aws_networkfirewall_firewall_policy": config.TemplatedStringAsIdentifier("name", "arn:aws:network-firewall:{{ .parameters.region }}:{{ .setup.client_metadata.account_id }}:firewall-policy/{{ .external_name }}"),
```